src/main/scala/com/fulcrumgenomics/sopt/cmdline/ClpArgumentDefinitionPrinting.scala

```diff
@@ -42,6 +42,15 @@ object ClpArgumentDefinitionPrinting {
   private[cmdline] val ArgumentDefaultValuePrefix: String = "Default:"
   private[cmdline] val ArgumentOptionalValue: String = "Optional"
 
+  /** A collection of non-printing ASCII characters and their string literals. */
+  private[cmdline] val NonPrintingCharacters: Map[String, String] = Map(
+    "\t" -> """\t""",
+    "\b" -> """\b""",
+    "\n" -> """\n""",
+    "\r" -> """\r""",
+    "\f" -> """\f""",
```

Comment:
should we really include "\f" and "b"?  Is there a need?  Why not also "\v"?

Reply:
I don't think `'\v'` exists as an escape sequence in Scala/Java land. I used the table for [Escape Sequences]( https://docs.oracle.com/javase/tutorial/java/data/characters.html) in the Oracle Java docs. I can surely delete some of them, but I listed them all out for the sake of completeness. You let me know what to do next.